fe/fe-core/src/main/java/com/starrocks/catalog/FileTable.java

```diff
@@ -129,9 +130,10 @@ public List<RemoteFileDesc> getFileDescsFromHdfs() throws DdlException {
         HiveRemoteFileIO remoteFileIO = new HiveRemoteFileIO(configuration);
         boolean recursive = Boolean.parseBoolean(fileProperties.getOrDefault(JSON_RECURSIVE_DIRECTORIES, "false"));
         RemotePathKey pathKey = new RemotePathKey(getTableLocation(), recursive, Optional.empty());
+        boolean enableWildCards = Boolean.parseBoolean(fileProperties.getOrDefault(JSON_ENABLE_WILDCARDS, "false"));
 
         try {
-            Map<RemotePathKey, List<RemoteFileDesc>> result = remoteFileIO.getRemoteFiles(pathKey);
+            Map<RemotePathKey, List<RemoteFileDesc>> result = remoteFileIO.getRemoteFiles(pathKey, enableWildCards);
```

Comment:
what's your current blocking point if we don't support wild cards? if you have a partitioned table, you could set path to table location and enable the `enable_recursive_listing`.

Reply:
oh, could you want to query multiple tables or partitions in a fileTable?